README.md

```diff
@@ -5,21 +5,32 @@
 log4j-sniffer
 ============
 
-log4j-sniffer pulls your archives apart looking for bad log4j versions.
+log4j-sniffer searches for all instances log4j that are earlier than version 2.16 on disk within a specified directory.
+It can be used to determine whether there are any vulnerable instances of log4j within a directory tree.
 
 What this does
 ==============
 
 log4j-sniffer will scan a filesystem looking for all files of the following types:
-- Zips: zip, par
-- Java archives: jar, war, ear
+- Zips: .zip, .par
```

Comment:
.par is an internal format, although it is just a zip I think we should likely drop it from the most public facing documentation to avoid confusion.